emu/src/memory/internal_memory.rs

```diff
@@ -151,7 +132,7 @@ impl IoDevice for InternalMemory {
             0x00000000..=0x00003FFF => self.bios_system_rom[address],
             0x02000000..=0x0203FFFF => self.working_ram[address - 0x02000000],
             0x03000000..=0x03007FFF => self.working_iram[address - 0x03000000],
-            0x04000000..=0x04000055 => self.lcd_registers.read_at(address),
+            0x04000000..=0x04000055 => unreachable!(),
```

Comment:
I think we can remove this... Since we should not go there

Reply:
You're right, thanks